Tests/SentryTests/PrivateSentrySDKOnlyTests.swift

```diff
@@ -18,7 +33,19 @@ class PrivateSentrySDKOnlyTests: XCTestCase {
         XCTAssertEqual(1, client?.storedEnvelopeInvocations.count)
         XCTAssertEqual(envelope, client?.storedEnvelopeInvocations.first)
     }
+    
+    func testStoreEnvelopeWithUndhandledMarksSessionAsCrashedAndDoesNotStartNewSession() {
```

Comment:
`l`: To slightly improve the readability:
```suggestion
    func testStoreEnvelopeWithUndhandled_MarksSessionAsCrashedAndDoesNotStartNewSession() {
```